specification/src/chapter2.adoc

```diff
@@ -2,9 +2,600 @@
+
+For the purpose of this specification a confidential service can be any isolated component on a system. For example, a hosted confidential workload, or an isolated application security service.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| A confidential service, and all software and hardware components it depends on, MUST be attestable. 
+|===
+
+Attestation allows a remote reliant party to determine the trustworthiness of a confidential service before submitting assets to it. 
+
+* Verify the security state of a confidential service
+* Verify the security state of all software and hardware a conidential service depends on
+* Establish an attested secure connection to a confidential service 
+
+Attestation can be direct or layered. 
+
+* Direct +
+The whole system can be defined by a single security platform attestation. For example, can be used in vertically integrated connected IoT devices and edge devices.
+* Layered +
+Enables parts of the attestation process to be delegated to lower privileged components.
+
+Direct and layered attestation are discussed in more detail in use case examples later in this specification.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| A security platform attestation MUST be signed by a HW RoT, if present, or by trusted security services
+
+| CAT_NNN
+| A security platform attestation MUST be signed using a hardware provisioned (directly or derived) secure identity
+
+| CAT_NNN
+| A layered attestation MAY be signed by lower privileged software, itself attested by a security platform attestation
+
+| CAT_NNN
+| A layered attestation MUST be signed by a secure identity cryptographically bound (for example, hash locked) to a fresh security platform attestation
+
+|===
+
+NOTE: Care needs to be taken in attestation interface design. For example, software interfaces should only support either direct attestation or layered attestation workflows, never both, to prevent impersonation.
+
+==== Authorized software 
+
+Running unauthorized software can compromise the security state of the system. Two complementary processes can be used to authorize software:
+
+* Measuring 
+* Verification
+
+A measurement is a cryptographic fingerprint of the launch state of a software component, such as a running hash of its memory contents and launch state.
+
+Verification compares an actual measurement to an expected measurement from a signed authorization by a trusted signer. Verification requires a provisioned or securely discovered list of trusted signer(s). Depending on use case, a signed authorization can be part of a software image. Or it can be delivered or discovered separately as part of an authorization protocol.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#     
+| Requirement
+
+| CAT_NNN  
+| All isolated software that has to be trusted by other software on the same system MUST be measured and verified.
```

Comment:
should verification be optional (may) since some platforms may find measured boot with attestation sufficient?

Reply:
I have updated the "Authorized software" section. Please review.